test/_utils.js

```diff
@@ -140,6 +140,28 @@ function sortApmEvents (events) {
     })
 }
 
+/**
+ * Slurp everything from the given ReadableStream and return the content,
+ * converted to a string.
+ */
+async function slurpStream (stream) {
```

Comment:
Note for reviewer: function copied here to be used in `use-client-*` files. Current `dynamodb` tests don't make use of it, yet. Update in `use-client-s3.js` pending